src/components/pages/Volunteers/index.tsx

```diff
@@ -146,19 +146,20 @@ const UnconnectedVolunteers: React.FC<PropsFromRedux> = ({
     </Container>
   );
 
+  let volunteers_id,
+    details_id = 'content';
   if (volunteers.loading) {
-    return spinner;
+    volunteers_id = 'faded';
+    details_id = 'faded';
   }
-
-  let page_id = 'content';
   if (volunteers.loading_details) {
-    page_id = 'faded';
+    details_id = 'faded';
   }
 
   return (
     <div className="d-flex flex-row">
       <section
-        id={page_id}
+        id={volunteers_id}
```

Comment:
ok for this to not require major refactoring, I'd just do
the content ID is confusing bc it's not doing anything

volunteers.loading ? 'faded' : ''